src/tests/gc-e2e/GuestConfiguration.Tests.ps1

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# GuestConfiguration.Tests.ps1
+param (
+    [Parameter(Mandatory)]
+    [string] $PolicyPackage
+)
+
+Describe 'Validate GuestConfiguration NRP' {
+    # Perform monitoring - Get
+    Context "Get" {
+        BeforeAll {
+            $result = Get-GuestConfigurationPackageComplianceStatus -Path $PolicyPackage -Verbose
```

Comment:
>Get-GuestConfigurationPackageComplianceStatus  [](http://example.com/codeflow?start=22&length=46)

"GuestConfiguration" is to remain only in the actual PS cmdlet names which we cannot change. These cmdlets were made before GC was renamed to AutoManage Machine Configuration.